openedx/core/djangoapps/discussions/admin.py

```diff
@@ -0,0 +1,35 @@
+from config_models.admin import KeyedConfigurationModelAdmin
+from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+
+from .models import DiscussionConfig, LearningContextDiscussionConfig
+
+
+class DiscussionConfigAdminModel(KeyedConfigurationModelAdmin):
+    search_fields = ("site", "org", "org_course", "context_key", "slug", "name", "description", "discussion_provider")
+    fieldsets = (
+        (_("Key settings"), {
+            "fields": ("enabled", "slug", "provider",),
+            "description": _("These setting should never be changed once set!"),
+        }),
+        (_("Name and Description"), {
+            "fields": ("name", "description"),
+            "description": _("The name and description can be visible to Course Authors, make them useful.")
+        }),
+        (_("Scope"), {
+            "fields": ("site", "org", "org_course", "context_key"),
+            "description":
+                _("The scope set here will define where all this configuration can be used."
+                  "If nothing is set the config is available globally. If only a site is set"
+                  "then the configuration is only available to learning contexts (courses "
+                  "for instance) in that site. Only of these can be set at a time."
+                  "DO NOT EDIT ONCE SET!")
```

Comment:
Shouldn't we enforce this?

Reply:
This admin UI is temporary. There will be a proper API and react UI for this, and this will go away. Enforcing this will currently require complicated Django admin code.